tests/Integration/DSC_SqlRSSetup.Integration.Tests.ps1

```diff
@@ -37,6 +37,7 @@ try
         $previousProgressPreference = $ProgressPreference
         $ProgressPreference = 'SilentlyContinue'
 
+        # TODO: Does this need to be different between SQL2017 and SQL2019?
```

Comment:
I'm unclear if this needs to be changed for SQL 2019? ... And if so, for/to what?